packages/app/src/embed/components/Files/index.js

```diff
@@ -15,7 +18,7 @@ import { Container } from './elements';
 
 type Props = {
   modules: Array<Module>,
-  directories: Array<Directory>,
+  directories: Array<DirectoryType>,
```

Comment:
You could write this as

```suggestion
  directories: DirectoryType[],
```